playground/backend/internal/environment/environment_service.go

```diff
@@ -240,6 +246,16 @@ func getConfigFromJson(configPath string) (*ExecutorConfig, error) {
 	return &executorConfig, err
 }
 
+// getDefaultExamplesPathFromJson reads a json file and returns default example path
+func getDefaultExamplesPathFromJson(configPath string) (string, error) {
+	file, err := ioutil.ReadFile(configPath)
+	if err != nil {
+		return "", err
+	}
+	defaultExamplePath := gjson.Get(string(file), defaultExampleKey).String()
+	return defaultExamplePath, nil
+}
```

Comment:
I believe, we already read such files with `Unmarshal`, can we do here the same to avoid adding a new dependency `gjson`?

Reply:
Done.